README.md

```diff
@@ -31,6 +31,7 @@ usage: redis-stat [HOST[:PORT] ...] [INTERVAL [COUNT]]
         --no-color                   Suppress ANSI color codes
         --csv=OUTPUT_CSV_FILE_PATH   Save the result in CSV format
         --es=ELASTICSEARCH_URL       Send results to ElasticSearch: [http://]HOST[:PORT][/INDEX]
+        --estags=ELASTICSEARCH_TAGS  List of comma seperated tags to add ro results sent to ElasticSearch: tag1,tag2
```

Comment:
- "add ro results"? Maybe a typo?
- The line is a bit to long. I think we can remove the trailing `: tag1,tag2` as it's already pretty obvious
- Let's rename the option to `--es-tags`